src/modules/Dropdown/Dropdown.js

```diff
@@ -255,20 +256,20 @@ export default class Dropdown extends Component {
   }
 
   closeOnEscape = (e) => {
-    if (e.key !== 'Escape') return
+    if (keyboardKey.getCode(e) !== keyboardKey.Escape) return
```

Comment:
Util includes `getCode` and `getName` methods, as well as constants for all keyCode/which codes by key name.